utils/hare-start

```diff
@@ -51,12 +51,12 @@ TEMP=$(getopt --options h \
 eval set -- "$TEMP"
 
 conf_dir=/var/lib/hare
-node=false
+node=
```

Comment:
This is how we use boolean in some of the scripts,
```
conf_create=false
opt_mkfs=

while true; do
    case "$1" in
        -h|--help)      usage; exit ;;
        -c|--conf-dir)  conf_dir=$2; shift 2 ;;
        --consul-addr)  CONSUL_ADDR=$2; shift 2 ;;
        --consul-port)  CONSUL_PORT=$2; shift 2 ;;
        --mkfs)         opt_mkfs=--mkfs; shift ;;
        --conf-create)  conf_create=true; shift ;;
        --)             shift; break ;;
        *)              break ;;
    esac
done
...
    if $conf_create; then
        say 'Generating node configuration...'
        PATH="$(dirname $(readlink -f $0)):$PATH" cfgen -o $conf_dir $cdf
        dhall text < $conf_dir/confd.dhall | m0confgen > $conf_dir/confd.xc
        while read node _; do
        if is_localhost $node; then
            echo $node > $conf_dir/node-name
        fi
        done < <(get_servers_from_conf)
        echo ' OK'
   fi
```

Reply:
Updated.